File: twiml/voice/connect/virtualagent/virtualagent.5.x.rb
require 'twilio-ruby'

response = Twilio::TwiML::VoiceResponse.new
response.connect(action: 'https://myactionurl.com/twiml') do |connect|
    connect.virtual_agent(connector_name: 'project', status_callback: 'https://mycallbackurl.com')
end

puts response

File: twiml/voice/connect/virtualagent-3/virtualagent.5.x.cs
using System;
using Twilio.TwiML;
using Twilio.TwiML.Voice;


class Example
{
    static void Main()
    {
        var response = new VoiceResponse();
        var connect = new Connect(action: new Uri("https://myactionurl.com/twiml"));
        connect.VirtualAgent(connectorName: "project", statusCallback: "https://mycallbackurl.com");
        response.Append(connect);

        Console.WriteLine(response.ToString());
    }
}
